index.html

```diff
@@ -21,18 +21,18 @@
           <table class="table">
             <tbody>
               <tr>
-                <td>Saturday, December 12</br>
-                  11:00 AM - 6:00 PM (EST)
+                <td>February 27, 2016</br>
```

Comment:
I think it's probably beneficial to list the day of the week. Otherwise this looks good to me!